archive/archive.go

```diff
@@ -85,7 +85,7 @@ func TarFileWriter(ctx context.Context, src, dst string, tw *tar.Writer, opts ..
 	}
 
 	if err := tw.WriteHeader(header); err != nil {
-		return fmt.Errorf("tar: %w", err)
+		return errors.Annotatef(err, "tar:%v", err)
```

Comment:
juju's error package can't handle %w format. So I try to implement in this way to have same effect.